generatebundlefile/data/input_120.yaml

```diff
@@ -7,13 +8,4 @@ packages:
         repository: eks-anywhere-test
         registry: public.ecr.aws/f5b7k4z5
         versions:
-            - name: v1.0.1-helm
-            - name: v0.1.1-latest
-            - name: latest
-  - org: jetstack
-    projects:
-      - name: cert-manager
-        repository: jetstack/cert-manager
-        registry: public.ecr.aws/f5b7k4z5
-        versions:
-            - name: v1.5.3-latest
+            - name: v0.1.1-latest #https://github.com/aws/eks-anywhere-build-tooling/blob/main/projects/aws/eks-anywhere-test/GIT_TAG
```

Comment:
Same thing for this version. This might still depend on other PRs though.

Reply:
Same thing fixed here https://github.com/aws/eks-anywhere-build-tooling/commit/a771eaa0ae98ba09b85ff0b8953c32d6f8b9fbc4